Balloon/sys/trace.h

```diff
@@ -28,10 +28,6 @@
  */
 #include "kdebugprint.h"
 
-#if ((OSVERSION_MASK & NTDDI_VERSION) > NTDDI_VISTA)
-#define EVENT_TRACING
```

Comment:
Please keep line 32. Otherwise EVENT_TRACING will not be defined.

Reply:
Added